Makefile

```diff
@@ -259,7 +259,6 @@ endif
 	$(MAKE) -C mlir format
 	$(MAKE) -C runtime format
 	$(MAKE) format-frontend
-	pylint frontend
```

Comment:
Is this intentional?

Reply:
Yes & no :D I removed it because `pylint` is kinda slow and not really formatting. I'll thus propose to keep the change, but if you mind I'll revert.

Reply:
Fair enough, if you use the format command often (rather than letting the ide do it), it can see how that could be cumbersome.  I'm okay with the change 👍